9.0/docker-entrypoint.sh

```diff
@@ -8,6 +8,7 @@ set_listen_addresses() {
 
 if [ "$1" = 'postgres' ]; then
 	mkdir -p "$PGDATA"
+	chmod -R 700 "$PGDATA"
```

Comment:
Does everything in the directory need to be `700` as well or should we drop the `-R`?